tests/integration/advanced/test_timeseries.py

```diff
@@ -1,7 +1,41 @@
+import importlib
+
 from tests.integration.helpers import ClickhouseTest, break_dataset
+from lightwood.api.types import ProblemDefinition
 from lightwood.api import make_predictor
 
 
 class TestTimeseries(ClickhouseTest):
     def test_timeseries(self):
```

Comment:
This doesn't need to inherit from `ClickhouseTest`

Also could we add a second test (or maybe even 2 x) one testing multiple predictions and one testing anomalies.
Also might be worth adding a text for predicting the "next" value (i.e. target in the following row instead of the current row) for when that is implemented.